orchestrator/orchestrator.go

```diff
@@ -228,7 +249,7 @@ func (o *Orchestrator) L1Chain() config.Chain {
 
 func (o *Orchestrator) L2Chains() []config.Chain {
 	var chains []config.Chain
-	for _, chain := range o.l2OpSims {
```

Comment:
@jakim929  this change was in main, and was causing linting issues, so updated it to l2Chain